diff --git a/setup.py b/setup.py
index aca2f88..b68935c 100644
--- a/setup.py
+++ b/setup.py
@@ -7,6 +7,6 @@ setup(
         author='Mariano Forti @ ICAMS',
         author_email = 'mariano.forti@icams.rub.de',
         description='a collection of scripts to collect high througput calculations',
-        install_requires=['pandas','numpy','tqdm', 'matplotlib', 'ase','pymatgen==2021.2.16','mendeleev'], 
+        install_requires=['pandas','numpy','tqdm', 'matplotlib', 'ase','pymatgen','mendeleev'], 
         include_package_data=True
         )
